### ISSUE TITLE
Tab from the background is brought to the top in Firefox

### ISSUE THREAD

**Issue by CennoxX** [cavi-au/Consent-O-Matic#486]
When I open a link in the background (for example through middle mouse click on a link) and a cookie-consent is automatically filled in the tab, I expect this to be done in the background.
Instead in some cases in Firefox the tab from the background is brought to the top, when the cookie-consent is confirmed/cannot be confirmed.

### Steps to Reproduce:
1. Middleclick on the link https://www.webtoffee.com/gdpr-cookie-consent-banner-examples/
2. Wait for 15 seconds
3. Focus will switch to the site
4. To retry, clear cookies of said site

Firefox 131.0.2 (64-Bit) (no problems with Chrome)
Microsoft Windows 10 Pro, Build 19045

**Comment by svnhub** [comment]
This works differently on Linux apparently, where the tab seems to stay in the background while being handled.
To be honest nobody thought about this specific use case - we will have to verify that the code even does what it is supposed to in this situation, not just in regard to the focus issue mentioned.

**Comment by Damocles-fr** [comment]
Yes I have the same issue on Firefox 138.0.4 (64 bits) Windows 11, on lot of websites

**Comment by Mncdk** [comment]
I just started using Consent-O-Matic, and I'm also experiencing this issue in Firefox.
